armory/utils/config_loading.py

```diff
@@ -220,34 +220,45 @@ def load_defense_internal(defense_config, classifier):
 
 
 def load_label_targeter(config):
-    scheme = config["scheme"].lower()
-    if scheme == "fixed":
-        value = config.get("value")
-        return labels.FixedLabelTargeter(value)
-    elif scheme == "string":
-        value = config.get("value")
-        return labels.FixedStringTargeter(value)
-    elif scheme == "random":
-        num_classes = config.get("num_classes")
-        return labels.RandomLabelTargeter(num_classes)
-    elif scheme == "round-robin":
-        num_classes = config.get("num_classes")
-        offset = config.get("offset", 1)
-        return labels.RoundRobinTargeter(num_classes, offset)
-    elif scheme == "manual":
-        values = config.get("values")
-        repeat = config.get("repeat", False)
-        return labels.ManualTargeter(values, repeat)
-    elif scheme == "identity":
-        return labels.IdentityTargeter()
-    elif scheme == "matched length":
-        transcripts = config.get("transcripts")
-        return labels.MatchedTranscriptLengthTargeter(transcripts)
-    elif scheme == "object_detection_fixed":
-        value = config.get("value")
-        score = config.get("score", 1.0)
-        return labels.ObjectDetectionFixedLabelTargeteer(value, score)
-    else:
-        raise ValueError(
-            f'scheme {scheme} not in ("fixed", "random", "round-robin", "manual", "identity", "matched length")'
+    if config.get("scheme"):
+        scheme = config["scheme"].lower()
```

Comment:
Can you add a "deprecation" warning on the logger?

Reply:
that's a good idea, added